README.md

```diff
@@ -60,6 +65,22 @@ In this case you can define a regular expression, which matches all user agent s
 ```
 This regular expression would match any user agent string, that ends with *My Example Webapp*.
 
+### Add a separate links specific to some device/platform
```

Comment:
Please move all these new sections to the end of _Advanced Usage_ to not to break the connected context of other advanced topics.